include/afl-fuzz.h

```diff
@@ -751,6 +767,17 @@ typedef struct afl_state {
    * is too large) */
   struct queue_entry **q_testcase_cache;
 
+  int  randomize_parameters_prob;
+
+  /* list of fuzzing parameter constants found in config.h */
```

Comment:
I think I spotted the bug here. The following parameters are being used without initialization when `AFL_DISABLE_RP` is set.

Reply:
I see. So, we should call `reset_fuzzing_params(afl)` even if `AFL_DISABLE_RP` is set?

Reply:
Currently no. This may explain why we have a very small number of executions.

Reply:
Yes, we can either call `reset_fuzzing_params` or initialize these variables with default values.